pimd/pim_pim.c

```diff
@@ -585,14 +585,70 @@ static int pim_msg_send_frame(int fd, char *buf, size_t len,
 		return pim_msg_send_frame(fd, (char *)ip2, sendlen, dst, salen,
 					  ifname);
 	}
-#endif
 
 	zlog_warn(
 		"%s: sendto() failure to %pSU: iface=%s fd=%d msg_size=%zd: %m",
 		__func__, dst, ifname, fd, len);
 	return -1;
 }
 
+#else
+static int pim_v6_msg_send(pim_addr src, pim_addr dst, ifindex_t ifindex,
```

Comment:
Till now, we have been keeping the function names same for pimv4 and pimv6 for cases like these. But now PIMv4 is using pim_msg_send_frame but PIMv6 pim_v6_msg_send.

Reply:
Ack